packages/firestore/test/util/helpers.ts

```diff
@@ -233,7 +232,7 @@ export function bound(
 ): Bound {
   const components: FieldValue[] = [];
   for (const value of values) {
-    const [field, dataValue] = value;
+    const dataValue = value[1];
```

Comment:
This is less readable. :-/  Is there any other option?  a placeholder instead of `field`?  What if you bump this into the for loop? (i.e. `for (const [field, dataValue] of values) { ... }`